package.json

```diff
@@ -20,39 +26,34 @@
     "component"
   ],
   "scripts": {
+    "dev": "vite dev",
```

Comment:
Yeah, I had no idea that we could use do `vite dev` here -- I agree and dig the update 👍